hazelcast/src/test/java/com/hazelcast/cache/CacheEntryViewsTests.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.cache;
+
+import com.hazelcast.cache.impl.CacheEntryViews;
+import com.hazelcast.cache.impl.record.CacheObjectRecord;
+import com.hazelcast.internal.serialization.SerializationService;
+import com.hazelcast.internal.serialization.impl.DefaultSerializationServiceBuilder;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link com.hazelcast.cache.impl.CacheEntryViews} class.
+ */
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class CacheEntryViewsTests extends HazelcastTestSupport {
+
+    @Test
+    public void testPrivateConstructor() {
+        HazelcastTestSupport.assertUtilityConstructor(CacheEntryViews.class);
+    }
+
+    @Test
+    public void testDefaultEntryViewCreation() {
+        String key = "testKey";
+        String value = "testValue";
+        CacheObjectRecord record = new CacheObjectRecord(value, System.currentTimeMillis(), Long.MAX_VALUE);
+        SerializationService serializationService = new DefaultSerializationServiceBuilder().build();
```

Comment:
To make your test more reliable, it is best not to set a well known value as Long.MAX_VALUE and verify that. Because it could be that due to a programming error the expiryTime is not set and defaults to LONG_MAX_VALUE. But you will never notice it.